Tensorflow 2.0

Installed Tensorflow 2.0 today and tried bert-as-service (BAS) which throws Tensorflow>=1.10 error. If tensorflow 2.0 doesn't break BAS, then update the assert to allow it. If it does break BAS, then documentation and error update to "Tensorflow >=1.10 and <2.0 required!"

bert-serving-start -model_dir /home/Downloads/cased_L-24_H-1024_A-16  -num_worker=1  -pooling_strategy NONE -max_batch_size 256 -gpu_memory_fraction 0.9
Traceback (most recent call last):
  File "/usr/local/bin/bert-serving-start", line 6, in <module>
    from bert_serving.server.cli import main
  File "/usr/local/lib/python3.6/dist-packages/bert_serving/server/__init__.py", line 29, in <module>
    _tf_ver_ = check_tf_version()
  File "/usr/local/lib/python3.6/dist-packages/bert_serving/server/helper.py", line 160, in check_tf_version
    assert int(tf_ver[0]) >= 1 and int(tf_ver[1]) >= 10, 'Tensorflow >=1.10 is required!'
AssertionError: Tensorflow >=1.10 is required!

**Prerequisites**
> Please fill in by replacing `[ ]` with `[x]`.

* [ x] Are you running the latest `bert-as-service`?
* [ x] Did you follow [the installation](https://github.com/hanxiao/bert-as-service#install) and [the usage](https://github.com/hanxiao/bert-as-service#usage) instructions in `README.md`?
* [ x] Did you check the [FAQ list in `README.md`](https://github.com/hanxiao/bert-as-service#speech_balloon-faq)?
* [ x] Did you perform [a cursory search on existing issues](https://github.com/hanxiao/bert-as-service/issues)?

--- a/client/bert_serving/client/__init__.py
+++ b/client/bert_serving/client/__init__.py
@@ -333,7 +333,7 @@ class BertClient(object):
             tmp = list(self.fetch())
             if sort:
                 tmp = sorted(tmp, key=lambda v: v.id)
-            tmp = [v.content for v in tmp]
+            tmp = [v.embedding for v in tmp]
             if concat:
                 if self.output_fmt == 'ndarray':
                     tmp = np.concatenate(tmp, axis=0)

--- a/server/bert_serving/server/helper.py
+++ b/server/bert_serving/server/helper.py
@@ -4,6 +4,7 @@ import os
 import sys
 import time
 import uuid
+import warnings
 
 import zmq
 from termcolor import colored
@@ -157,7 +158,11 @@ def get_args_parser():
 def check_tf_version():
     import tensorflow as tf
     tf_ver = tf.__version__.split('.')
-    assert int(tf_ver[0]) >= 1 and int(tf_ver[1]) >= 10, 'Tensorflow >=1.10 is required!'
+    if int(tf_ver[0]) <= 1 and int(tf_ver[1]) < 10:
+        raise ModuleNotFoundError('Tensorflow >=1.10 (one-point-ten) is required!')
+    elif int(tf_ver[0]) > 2:
+        warnings.warn('Tensorflow %s is not tested! It may or may not work. '
+                      'Feel free to submit an issue at https://github.com/hanxiao/bert-as-service/issues/' % tf.__version__)
     return tf_ver
 
 

--- a/server/bert_serving/server/helper.py
+++ b/server/bert_serving/server/helper.py
@@ -160,7 +160,7 @@ def check_tf_version():
     tf_ver = tf.__version__.split('.')
     if int(tf_ver[0]) <= 1 and int(tf_ver[1]) < 10:
         raise ModuleNotFoundError('Tensorflow >=1.10 (one-point-ten) is required!')
-    elif int(tf_ver[0]) > 2:
+    elif int(tf_ver[0]) > 1:
         warnings.warn('Tensorflow %s is not tested! It may or may not work. '
                       'Feel free to submit an issue at https://github.com/hanxiao/bert-as-service/issues/' % tf.__version__)
     return tf_ver

